src/app/item-popup/DesktopItemActions.tsx

```diff
@@ -28,9 +28,8 @@ import LockButton from './LockButton';
 
 export default function DesktopItemActions({ item }: { item: DimItem }) {
   const stores = useSelector(sortedStoresSelector);
-  const settings = useSelector(settingsSelector);
+  const { sidecarCollapsed } = useSelector(settingsSelector);
```

Comment:
It's better to do `const sidecarCollapsed = useSelector((state) => settingsSelector(state).sidecarCollapsed)`. What you have will re-render any time settings changes, while what I wrote will re-render only when `sidecarCollapsed` changes.

```suggestion
  const { sidecarCollapsed } = useSelector(settingsSelector);
```